jobs/integr8ly/ocp3/test-suites/w1-test-executor.yaml

```diff
@@ -38,7 +38,15 @@
       - string:
           name: TIMEOUT_THRESHOLD
           default: '1'
-          description: 'optionally increase timeout values. If you provide value 2 it means it will be sleeping/waiting two times longer'      
+          description: 'optionally increase timeout values. If you provide value 2 it means it will be sleeping/waiting two times longer'
+      - bool:
+          name: W1_FINAL_VERIFICATION
```

Comment:
Should not this be always set to true for this w1-test pipeline? IMO `Verify W1A works as expected` step should be performed every time.

Reply:
You are right. I changed it